setup.cfg

```diff
@@ -0,0 +1,24 @@
+[flake8]
+exclude = .venv,.git,.tox,docs,venv,bin,lib,deps,build
+# To work with Black
+max-line-length = 88
+# D202: No blank lines allowed after function docstring
+# D401: First line should be in imperative mood
+# E203: Whitespace before ':'
+# E501: line too long
```

Comment:
Why do you ignore E501 when you could just set `max-line-length` above to what is currently the longest line (plus some margin)?

Reply:
Closing PR since it's now obsolete.